.travis.yml

```diff
@@ -6,7 +6,6 @@ python:
   - 3.4
   - 3.5
   - 3.6
-  - nightly
```

Comment:
Looking at the build history, this seems to be flaky. I'm not sure that there's much value in keeping it.